index.js

```diff
@@ -344,6 +349,9 @@ class WebdriverAjax {
     // (best effort compliance with RFC)
     function parseResponseHeaders(rawHeader) {
       const headers = {};
+      if (rawHeader === '') {
```

Comment:
`XMLHttpRequest.getAllResponseHeaders()` can return `null` in exceptional cases
```suggestion
      if (!rawHeader) {
```